Client/core/CMainMenu.h

```diff
@@ -93,10 +93,18 @@ class CMainMenu
     bool OnQuitButtonClick(CGUIElement* pElement);
     bool OnNewsButtonClick(CGUIElement* pElement);
 
+    bool OnSocialMediaButtonClick(CGUIElement* pElement);
+    bool OnSocialMediaButtonHover(CGUIElement* pElement);
+    bool OnSocialMediaButtonUnhover(CGUIElement* pElement);
+
     void HideServerInfo();
 
     CGUI* m_pManager;
 
+    CGUIStaticImage* m_socialButtons[6] = {nullptr};
+    std::map<CGUIStaticImage*, const char*> m_socialButtonLinks;
+    std::map<const char*, const char*> m_socialMediaLinks;
```

Comment:
These two std::map are unnecessary and the static images were leaked. Here is the replacement:
```cpp
std::array<std::unique_ptr<CGUIStaticImage>, 6> m_socialsImages{};
```